models/movies/detailed.js

```diff
@@ -0,0 +1,31 @@
+
+// Handle Upcomings Movies
+
+var url = "https://api.themoviedb.org/3/movie/" + movie_id + "?api_key=" + key + "&language=en-US&page=1";
+
+AJAX.request(url,function() {
+    if (!result.status_code) {
```

Comment:
Shouldn't this be something like this?
```javascript
if (result.status_code === 200) {
  //remove loading
  //render my movie card
}
```

Reply:
Not really. "status_code" it is the API's Error Code. The value came when you got an error.. BUT check for server response status is missing and its really important.